helix-core/src/main/java/org/apache/helix/task/JobContext.java

```diff
@@ -313,4 +313,12 @@ public long getExecutionStartTime() {
     }
     return map;
   }
+
+  /**
+   * Remove the task from the map field of job context.
+   * @param taskNumber
+   */
+  public void removeTask(int taskNumber) {
```

Comment:
nit, taskSeqNumber?

Reply:
Moreover, according to the naming convention of this class, I guess we are using "partition" instead of "task".

Reply:
Done.